### DISCUSSION TITLE
Avatar - single, double letter, image, allow picking alignment (left/right)

### DISCUSSION THREAD

**Discussion by Androrier** [flyerhq/flutter_chat_ui#106] [Ideas & Requests]
Which will include the first letter from first name and the first letter from last name?
Today as I see, it's only the first letter of the first name.
What do you think?

**Comment by demchenkoalex** [comment]
I think we can check if only `firstName` is set we set 1 letter and if both are set we set 2, makes sense?

**Reply by Androrier** [reply]
Yes, thats great
